### ISSUE TITLE
keyboard shortcut for annotations conflicts with the web developer tools in Windows

### ISSUE THREAD

**Issue by amirsol81** [edrlab/thorium-reader#2825]
Thorium Reader has two hotkeys for annotation creation when text is selected:
1. Control+ALT+SHIFT+A, which opens a dialog for more text options.
2. Control+ALT+SHIFT+Q, which quickly creates an annotation.
However, in the case of the first one, Thorium Reader erroneously opens the Developer Tools window. Actually two windows which should be closed with ALT+F4 to return to the book.
And in the case of the second one, the quick annotation creation option, Thorium Reader just says the following: Alert no text selected.
Note that text is indeed selected, from a word to several words, for both cases.
This is on Windows 11 24H2 with NVDA.

**Comment by danielweck** [comment]
>  Note that text is indeed selected, from a word to several words, for both cases.

For the avoidance of doubt, the text is in fact not selected inside the webview that displays the HTML document (standard selection DOM Range), instead the selection only exists in the closed proprietary world of the screen reader. This is a known issue with screen readers in general. NVDA is working on a solution for Chromium which is the web browser engine used in Thorium (same as Google Chrome, Microsoft Edge, etc. etc.): https://github.com/edrlab/thorium-reader/issues/2814#issuecomment-2690326743

**Comment by danielweck** [comment]
As for the keyboard shortcut problem: this bug is reported for Windows, doesn't affect MacOS, not sure about Linux. Essentially, the combination of keys is already used by the Web Developer tools in Chromium, and it seems to take precedence over Thorium's own keyboard shortcut. Note that the developer tools are not activated in official releases, only in test builds. Still, this breaks the annotations workflow and needs to be fixed or worked around.

**Comment by nvdaes** [comment]
@danielweck , what about adding available features, at least some of them like bookmarks and annotations, via the context menu (shift+f10 on Windows)? The context menu exists noe and an option to inspect is presented. Perhaps adding bookmarks and annotations to the menu will also solve other issues with screen readers like knowing the corrent position if a click event is triggered.

**Comment by nvdaes** [comment]
Also, this may make Thorium more accessible for people with dexterity difficulties.

**Comment by nvdaes** [comment]
@danielweck , I think that the selected paragraph is not shown, or I don't find it in the inspector. I find a toolbar relater to synthesizer, a list etc.

**Comment by danielweck** [comment]
Hello, I was able to reproduce this with the META key instead of CONTROL on MacOS. 
The bookmark creation keyboard shortcut is `CONTROL` + `B` (actually toggles the current bookmark if there is one at the exact current reading location) and the variant that immediately shows the comment / note editor is `SHIFT` + `ALT` + `B`. For creating annotations it is `SHIFT` + `ALT` + `A`, or its counterpart `SHIFT` + `ALT` + `Q` which is for quick creation without the input dialog for comment / note, colour, etc.

Important note: please reset the keyboard shortcuts to factory defaults in Thorium's preferences from the main library window, otherwise it is likely that there will be some persistence of previously-configured options.

Full discussion:

https://github.com/edrlab/thorium-reader/discussions/2867#discussioncomment-12544582
